.docforge/manifest.yaml

```diff
@@ -1,9 +1,21 @@
-# SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and Gardener contributors
-#
-# SPDX-License-Identifier: Apache-2.0
```

Comment:
Can we keep the license header?

Reply:
Sure. Stupid question maybe, what's the background/why?

I found it only in 2 places I believe and since these are our files, I wasn't sure what's the benefit.

Reply:
Due to some standards ( I guess [reuse](https://reuse.software/)) or maybe licensing, we are required to have the license header on all possible files.

Reply:
OK, I added the folder to the reuse manifest, so that we can remove these less than helpful headers. `reuse lint` isn't showing any errors anymore. However, if `reuse` compliance is intended, there are some unrelated (to this PR unrelated) issues:
```
* Files with copyright information: 5732 / 7791
* Files with license information: 488 / 7791

Unfortunately, your project is not compliant with version 3.0 of the REUSE Specification :-(
```